README.md

```diff
@@ -1 +1,229 @@
-# Personal-Catalog
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+<a name="readme-top"></a>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Catalog of my things ](#-catalog-of-my-things-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+    - [Deployment](#deployment)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [❓ FAQ ](#-faq-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Catalog of my things <a name="about-project"></a>
+
+
+This project involves creating a console application that allows users to keep track of their personal collection of books, music albums, and games. The program is structured according to the [UML class diagram](./src/images/catalog_of_my_things.png). The data is stored in JSON files and a database is also set up with tables that match the class structure of the program.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Server Side</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+    <li><a href="https://www.json.org/">Json</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **List all books**
+- **List all music albums**
+- **List of movies**
+- **List of games**
+- **List all genres (e.g 'Comedy', 'Thriller')**
+- **List all labels (e.g. 'Gift', 'New')**
+- **List all authors (e.g. 'Stephen King')**
+- **List all sources (e.g. 'From a friend', 'Online shop')**
+- **Add a book**
+- **Add a music album**
+- **Add a movie**
+- **Add a game**
+- **Preserve data in JSON files**
+- **Created a console app**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Video Presentation](https://drive.google.com/file/d/1o7An3uENnyvTAp3Xm7TA56IB5Q4zfh3i/view?usp=share_link)
+
```

Comment:
- [ ] As the previous reviewer asked, Kindly reduce your video length and the maximum length of the video should be 5 minutes. Currently, the video length is 6 mins. :+1: